espnet/nets/pytorch_backend/decoders.py

```diff
@@ -163,7 +163,7 @@ def forward(self, hs_pad, hlens, ys_pad, strm_idx=0):
             if i > 0 and random.random() < self.sampling_probability:
                 logging.info(' scheduled sampling ')
                 z_out = self.output(z_all[-1])
-                z_out = np.argmax(z_out.detach(), axis=1)
+                z_out = np.argmax(z_out.detach().cpu(), axis=1)
```

Comment:
Do you need this?

Reply:
Error occurred when using the original code in the current master branch.